mmpretrain/datasets/textvqa.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from collections import Counter
+from typing import List
+
+import mmengine
+from mmengine.dataset import BaseDataset
+
+from mmpretrain.registry import DATASETS
+
+
+@DATASETS.register_module()
+class TextVQA(BaseDataset):
+    """TextVQA dataset.
+
+    val image:
+        https://dl.fbaipublicfiles.com/textvqa/images/train_val_images.zip
+    test image:
+        https://dl.fbaipublicfiles.com/textvqa/images/test_images.zip
+    val json:
+        https://dl.fbaipublicfiles.com/textvqa/data/TextVQA_0.5.1_val.json
+    test json:
+        https://dl.fbaipublicfiles.com/textvqa/data/TextVQA_0.5.1_test.json
+
```

Comment:
Please add folder structure of the textvqa dataset.

Reply:
done